rspec-mode.el

```diff
@@ -646,14 +667,24 @@ file if it exists, or sensible defaults otherwise."
 
 (defun rspec--shell-quote-local (file)
   (let ((remote (file-remote-p file))
+        (docker (rspec-docker-p))
         (vagrant (rspec-vagrant-p)))
     (shell-quote-argument
      (cond
       (remote (substring file (length remote)))
+      (docker (replace-regexp-in-string (regexp-quote (rspec-project-root))
+                                         rspec-docker-cwd file))
       (vagrant (replace-regexp-in-string (regexp-quote (rspec-project-root))
                                          rspec-vagrant-cwd file))
       (t  file)))))
 
+(defun rspec--docker-wrapper (command)
+  (if (rspec-docker-p)
+      (format "docker exec %s bash -c \"%s\""
```

Comment:
Would it be better to use `docker-compose`? It is more flexible regarding supporting services, etc. I know that one can `docker exec` into a composed service but it using compose directly would probably give us more auto starting options.